task/redhat-codeready-dependency-analysis/0.1/README.md

```diff
@@ -0,0 +1,116 @@
+# `redhat-codeready-dependency-analysis`
+**Please Note: this Task is only compatible with Tekton Pipelines versions 0.19.0 and greater!**
+
+## Overview
+The redhat-codeready-dependency-analysis task is an interface between Tekton and Red Hat CodeReady Dependency Analytics platform. 
+It provides vulnerability and compliance analysis for your applications dependencies, along with recommendations to address security vulnerabilities and licensing issues.
+
+This task reflects [Dependency Analytics VS Code plugin](https://marketplace.visualstudio.com/items?itemName=redhat.fabric8-analytics) for Tekton Pipelines.
+
+**Please Note: Currently this Task supports Python ecosystem, support for other ecosystems will be provided very soon.**
+
+## Prerequisite
+
+#### 1. Creation of Workspace
+Workspace is used as a common filesystem between tasks and used for inputs and outputs of task, in workspace dependencies are installed for further usage while running the task.
+
+This [sample](../0.1/samples/workspace.yaml) can be referred to create a workspace.<br />
+
+#### 2. Manifest File
+This task scans manifest file (ex. requirements.txt, requirements-dev.txt etc.) for collecting a list of dependencies, hence prior to this task, target manifest file must be placed by user into workspace. 
+The path to manifest is passed as parameters to task.
+
+#### 3. Dependency Installation
+Users need to install dependencies from manifest into a directory of workspace. 
+Location of installation directory in workspace is required as parameter in the task.
+
+#### 4. Adding Secret
+To authenticate a user this task uses CRDA user key. 
+This key should be saved in secrets, but before that users need to generate it.<br />
+This is a one time activity to perform. 
+
+Here are the steps to generate a CRDA user key.<br />
+1. Install CRDA CLI from [here](https://github.com/fabric8-analytics/cli-tools/releases).
+2. Run `crda auth` command, and it will assign user a unique id, which can be found in `~/.crda/config.yaml`. 
+Users need to store the value of `crda_key` into a `Secret` to run the task.
+<br />
+This [sample](../0.1/samples/secret.yaml) can be referred to create a secret file, replace `{{ CRDA_USER_KEY }}` with the generated CRDA key before running.
+<br />
+Following command can be used to create workspace from the file.
```

Comment:
```suggestion

This [sample](../0.1/samples/secret.yaml) can be referred to create a secret file, replace `{{ CRDA_USER_KEY }}` with the generated CRDA key before running.

Following command can be used to create workspace from the file.
```

Reply:
here <br /> is used because next line is under scope of subheading, if just a new line is given, it will shifts to left most margin of the page and will look like a new paragraph